workers/unity/Packages/io.improbable.gdk.gameobjectcreation/Tests/Editmode/GameObjectCreationTests.cs

```diff
@@ -23,8 +25,15 @@ protected override MockWorld.Options GetOptions()
             {
                 AdditionalSystems = world =>
                 {
+                    // TODO: Hacked in EntityLinkerDatabase
                     var testGameObjectCreator = new TestGameObjectCreator(WorkerType);
-                    GameObjectCreationHelper.EnableStandardGameObjectCreation(world, testGameObjectCreator);
+                    var dummyDatabase = ScriptableObject.CreateInstance<EntityRepresentationMapping>();
+                    dummyDatabase.EntityRepresentationResolvers = new List<IEntityRepresentationResolver>
+                    {
+                        new SimpleEntityResolver("TestObject", new GameObject())
```

Comment:
Should we add tests to cover all the resolvers we include in the GDK?

Reply:
I think those are useless tests